.github/workflows/EnforceLabels.yml

```diff
@@ -0,0 +1,19 @@
+name: Enforce PR labels
+
+permissions:
+  contents: read
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, reopened, edited, synchronize]
+jobs:
+  enforce-labels:
+    name: Check for blocking labels
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+    - uses: yogevbd/enforce-label-action@2.2.2
+      with:
+        REQUIRED_LABELS_ANY: "release notes: added,release notes: highlight,release notes: not needed,release notes: to be added,release notes: use title"
+        REQUIRED_LABELS_ANY_DESCRIPTION: "Select at least one label concerning release notes"
+        BANNED_LABELS: "breaking,DO NOT MERGE,needs tests,WIP"
```

Comment:
if we ban the `breaking` label, what would be the workflow for merging a breaking change then?

Reply:
Since we don't plan for a breaking release in the foreseeable future, IMO a PR with a `breaking` label shouldn't be merged until then. And this list of banned labels could then be adapted once we changed master to version `2.0.0-DEV`.

Furthermore, we have a handful of people who can merge, bypassing the CI restrictions, this would certainly be an option.

But that's just my opinion. If you disagree, please just remove this one label, so we don't block the rest of the PR over it.